packages/mux-player/src/styles.css

```diff
@@ -89,3 +89,9 @@ media-volume-range {
 media-time-display {
   white-space: nowrap;
 }
+
+.mxp-live-indicator::before {
+  content: "● ";
```

Comment:
non-blocking: 👍 any sizing weirdness as a result of `::before` instead of including it in the DOM structure?

Reply:
not really, did you expect some weirdness with using `::before`?